test/module/irohad/ametsuchi/block_query_test.cpp

```diff
@@ -389,3 +397,47 @@ TEST_F(BlockQueryTest, HasTxWithInvalidHash) {
   shared_model::crypto::Hash invalid_tx_hash(zero_string);
   EXPECT_FALSE(blocks->hasTxWithHash(invalid_tx_hash));
 }
+
+/**
+ * @given block store with preinserted blocks
+ * @when getTopBlock is invoked on this block store
+ * @then returned top block's height is equal to the inserted one's
+ */
+TEST_F(BlockQueryTest, GetTopBlockSuccess) {
+  boost::optional<shared_model::interface::types::HeightType> top_block_height;
+  blocks->getTopBlock().match(
```

Comment:
Please use methods from result_fixture to simplify the code

Reply:
Also we need to verify that returned block is indeed the top one

Reply:
The top-most attribute of the block is verified in 418 line via height comparison, which is unique for both blocks in this store, and only the top-most is of height 2